pkg/html/generichtml/errors.go

```diff
@@ -0,0 +1,36 @@
+package generichtml
+
+import (
+	"html/template"
+	"log"
+	"net/http"
+)
+
+var messageTemplate = template.Must(template.New("jobs").Parse(`<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>{{.StatusCode}} {{.StatusMessage}}</title>
+</head>
+<body>
+  <h1>{{.StatusCode}} {{.StatusMessage}}</h1>
+  <hr>
+  <p>{{.UserMessage}}</p>
+</body>
+</html>
+`))
+
+// PrintStatusMessage is used to display a proper error page to an end user.
+func PrintStatusMessage(w http.ResponseWriter, code int, message string) {
+	w.WriteHeader(http.StatusNotFound)
+	w.Header().Set("Content-Type", "text/html;charset=UTF-8")
+
+	err := messageTemplate.Execute(w, map[string]interface{}{
+		"StatusCode":    code,
+		"StatusMessage": http.StatusText(code),
+		"UserMessage":   message,
+	})
+	if err != nil {
+		log.Print(err)
```

Comment:
I noticed that Sippy uses [klog](https://github.com/kubernetes/klog) for logging, so this should probably be `klog.Errorf(err.Error())` (or similar).

Reply:
Ah thanks!